app/code/Magento/Theme/Block/Html/Header/Logo.php

```diff
@@ -101,6 +103,20 @@ public function getLogoWidth()
         return (int)$this->_data['logo_width'];
     }
 
+    /**
+     * Retrieve logo width
+     *
+     * @return int
+     */
+    public function getLogoImageWidth() :int
```

Comment:
should we change it to getLogoImageWidth(): int instead of getLogoImageWidth() :int

Reply:
As far as I know - the standard says about `: int`.

Reply:
@lbajsarowicz  is right and its mentioned in the PSR-12
![image](https://user-images.githubusercontent.com/39480008/91398792-68224b00-e85a-11ea-8b30-34b7e116eda8.png)
For more information :- https://www.php-fig.org/psr/psr-12/
But I don't know that we started following the PSR-12 standards?

Reply:
PSR-12 is in the works: https://github.com/magento/architecture/issues/225 & https://github.com/magento/magento-coding-standard/issues/128